pkg/http/api.go

```diff
@@ -1879,11 +1885,20 @@ func (a *api) onShutdown(reqCtx *fasthttp.RequestCtx) {
 }
 
 func (a *api) onPublish(reqCtx *fasthttp.RequestCtx) {
+	a.onPubSubPublish(reqCtx, "", "")
+}
+
+func (a *api) onActorPublishAlpha1(reqCtx *fasthttp.RequestCtx) {
+	actortype := reqCtx.UserValue(actorTypeParam).(string)
```

Comment:
nit: Maybe we can keep the original method and check for the presence of actorTypeParam and actorIdParam.